walk-in-interview/src/main/webapp/homepage/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="../style.css">
+</head>
+
+<!-- The content and element attributes for this file will be
+    rendered dynamically from the script.js file. -->
+
+<body>
+  <div class="header-container">
+    <button id="homepage-new-post"></button>
+    <div id="homepage-page-title"></div>
+    <button id="homepage-account"></button>
+  </div>
+
+  <div class="content-container">
+    <div id="homepage-map">map here</div>
```

Comment:
Is `map here` a temporary place holder or it is a text to be displayed here?

Reply:
temporary placeholder - I'll add a comment